packages/devtools_app/lib/src/config_specific/server/_server_web.dart

```diff
@@ -250,6 +250,49 @@ Future<void> resetDevToolsFile() async {
   }
 }
 
+Future<DevToolsJsonFile> requestPrimaryAppSizeFile(String path) async {
+  if (isDevToolsServerAvailable) {
+    final resp = await _request(
+      '$apiGetPrimaryAppSizeFile?$primaryAppSizeFilePropertyName=$path',
```

Comment:
Specify these as real query parameters instead of building the uri manually for more robust escaping. For example, what would happen if `path` contained an `&`

Reply:
The HTTP request fails when the path is encoded. 
e.g.
`http://localhost:9101/api/getTestAppSizeFile?appSizeTest=Downloads%2Fapk-size-analysis-encryption-app-new.json 40`

Reply:
Never mind - this was a naming issue, not due to encoding 👍